doc/Analytics.xml

```diff
@@ -3049,6 +3049,480 @@
+              </row>
+            </thead>
+
+            <tbody>
+              <row>
+                <entry><para>&lt;xmltag/&gt;</para></entry>
+
+                <entry><para>“xmltag”: null</para></entry>
+
+                <entry><para>Null tag</para></entry>
+              </row>
+
+              <row>
+                <entry><para>&lt;xmltag&gt;text&lt;/xmltag&gt;</para></entry>
+
+                <entry><para>“xmltag”: "text"</para></entry>
+
+                <entry><para>Simple tag with value</para></entry>
+              </row>
+
+              <row>
+                <entry><para>&lt; xmltag name="value" /&gt;</para></entry>
+
+                <entry><para>“xmltag”:{"@name": "value"}</para></entry>
+
+                <entry><para>Tag with attribute</para></entry>
+              </row>
+
+              <row>
+                <entry><para>&lt;xmltag name="value"&gt;text&lt;/
+                xmltag&gt;</para></entry>
+
+                <entry><para>“xmltag”: { "@name": "value", "#text": "text"
+                }</para></entry>
+
+                <entry><para>Tag with attribute and value</para></entry>
+              </row>
+
+              <row>
+                <entry><para>&lt;xmltag&gt; &lt;tag1&gt;text&lt;/tag1&gt;
+                &lt;tag2&gt;text&lt;/tag2&gt; &lt;/xmltag</para></entry>
+
+                <entry><para>“xmltag”: { "tag1": "text", "tag2": "text"
+                }</para></entry>
+
+                <entry><para>Tag with multiple child tags</para></entry>
+              </row>
+
+              <row>
+                <entry><para>&lt;xmltag&gt; &lt;tag1&gt;text&lt;/tag1&gt;
+                &lt;tag1&gt;text&lt;/tag1&gt; &lt;/ xmltag&gt;</para></entry>
+
+                <entry><para>“xmltag”: { "tag1": ["text", "text"]
+                }</para></entry>
+
+                <entry><para>Tag with multiple child tags of same type
+                (similar to minOccurs &gt; 1)</para></entry>
+              </row>
+            </tbody>
+          </tgroup>
+        </table>
+
+        <para>All extension elements and attributes shall be included within
+        the same parent JSON object.</para>
+
+        <para>Namespace prefixes for ONVIF defined namespaces shall be
+        dropped, i.e. elements and attributes that belong to the following
+        namespaces:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para>http://www.onvif.org/ver10/schema</para>
+          </listitem>
+
+          <listitem>
+            <para>http://www.onvif.org/ver20/analytics/humanface</para>
+          </listitem>
+
+          <listitem>
+            <para>http://www.onvif.org/ver20/analytics/humanbody</para>
+          </listitem>
+
+          <listitem>
+            <para>http://www.onvif.org/ver20/analytics/radiometry</para>
+          </listitem>
+        </itemizedlist>
+
+        <para>XML elements and attributes that belong to a different namespace
```

Comment:
Strongly propose to replace proprietary namespace handling by reference to context and prefix definition of [JSON-LD](https://www.w3.org/TR/json-ld11/).

Reply:
As discussed in Telco, json-ld is referred via the 'context' element, So no change needed.